dependencies.yaml

```diff
@@ -74,11 +83,11 @@ dependencies:
           - cxx-compiler
           - cython>=3.0.0
           - &doxygen doxygen==1.9.1
-          - libraft-headers==24.8.*
-          - librmm==24.8.*
+          - libraft-headers==24.8.*,>=0.0.0a0
+          - librmm==24.8.*,>=0.0.0a0
           - nanobind>=0.2.0
           - nccl
-          - scikit-build-core>=0.7.0
+          - &scikit_build_core_conda scikit-build-core>=0.7.0
```

Comment:
Is this needed here, or should it be expressed by adding the `rapids_build_skbuild` list key in the appropriate files? It seems odd that this would require `scikit-build-core` in the `build` list but not `rapids-build-backend`.

If you remove this, you can also get rid of the anchor name.

Reply:
You're totally right, this could just be removed.

I just pushed https://github.com/rapidsai/wholegraph/pull/181/commits/4112bf728b741383b308c1b6556cedbc76d999df doing that and adding `rapids_build_skbuild` to the one other file (`clang_tidy:`) that was using `build`.